src/postamble_modularize.js

```diff
@@ -11,10 +11,7 @@ if (runtimeInitialized)  {
   moduleRtn = Module;
 } else {
   // Set up the promise that indicates the Module is initialized
-  moduleRtn = new Promise((resolve, reject) => {
-    readyPromiseResolve = resolve;
-    readyPromiseReject = reject;
-  });
+  moduleRtn = readyPromise = createReadyPromise();
```

Comment:
Can we just do `moduleRtn = readyPromise` here and unconditionally call `createReadyPromise` in `runtime_common.js
`?